src/components/Modals/SetAclPermissionsModal.jsx

```diff
@@ -39,16 +40,27 @@ import useNotification from '../../hooks/useNotification';
 const SetAclPermissionsModal = ({ showModal, setShowModal, dataset }) => {
   const { session } = useSession();
   const { addNotification } = useNotification();
-  const { podUrl } = useContext(SignedInUserContext);
+
+  const { podUrl, webId } = useContext(SignedInUserContext);
+
   const [permissionState, setPermissionState] = useState({
-    podUrlToSetPermissionsTo: '',
+    webIdToSetPermsTo: '',
     permissionType: ''
   });
   const [processing, setProcessing] = useState(false);
 
+  const { data } = useContactsList();
+  const contactListOptions = data?.map((contact) => ({
+    label: `${contact.person} ${contact.webId}`,
+    id: contact.webId
+  }));
+  const shareeName = data?.filter(
```

Comment:
Double `e` in variable name, should be `shareName`

Reply:
That was intentional, though I'm not sure if sharee is _actually_ word haha. Meant it as the person being shared with. For readability though I'll make it shareName either way.

Reply:
I see, sharee like mentee, got it. No problem. 👍